scripts/qa_crowbarsetup.sh

```diff
@@ -2893,10 +2899,19 @@ function custom_configuration
                 local maybe_direct_interface=''
                 [[ $deployswift ]] && maybe_direct_interface=", 'direct'"
                 proposal_set_value ironic default "['attributes']['ironic']['enabled_deploy_interfaces']" "['iscsi' $maybe_direct_interface]"
+
+                if [[ $want_tempest ]]; then
```

Comment:
I guess what we actually want is a condition on something like $cloud =~ virtual as well..

Reply:
I don't think it's needed. Enabling "fake" driver just loads some additional modules. It doesn't make you use this driver if you want different one. And tempest seems to use fake driver for most of it's tests (see: https://github.com/openstack/ironic-tempest-plugin/blob/master/ironic_tempest_plugin/tests/api/admin/base.py#L28). I haven't even tested it with any "real" driver. If we want to go in that direction, we would need to add way more code to prepare the env for tests.

Reply:
wow, the upstream testing is more embarassing than I thought.